soccer/gameplay/tactics/pass_to_point_heuristic.py

```diff
@@ -0,0 +1,12 @@
+import constants
```

Comment:
If the thought is to have tons of these heuristics, somewhere else besides tactics might be a good idea. Evaluation is getting big as a folder, but that's always a possible place to put it

Reply:
These heuristics don't really mean anything outside of tactics, though. We could always make an entire `tactics.pass` directory and put them in there (and the actual tactic itself).

Reply:
What about a putting it under `heurustics/pass`?